src/components/MentorLandingPage/Reducer/MentorLandingPageReducer.js

```diff
@@ -0,0 +1,14 @@
+const carsReducer = (state = [], action) => {
```

Comment:
carReducer? Currently, the reducer isn't and will not doing anything. "CREATE_MENTOR" will return undefine and redux will not allow you to return undefined state. You need to establish what you are trying to get in the reducer. Is it an object? An array? If so, what key: values are you trying to get it to return?